ipaserver/plugins/user.py

```diff
@@ -603,11 +603,14 @@ def post_callback(self, ldap, dn, entry_attrs, *keys, **options):
             if not options.get('all', False):
                 desc_attr = ldap.get_entry(dn, ['description'])
                 entry_attrs.update(desc_attr)
+                entry_attrs._orig['description'] = list(
+                    desc_attr['description'])  # The entry_attrs.update()
+                # does not change the entry_attrs._orig
             if 'description' in entry_attrs and NO_UPG_MAGIC in entry_attrs['description']:
                 entry_attrs['description'].remove(NO_UPG_MAGIC)
-                kw = {'setattr': unicode('description=%s' % ','.join(entry_attrs['description']))}
                 try:
-                    self.api.Command['user_mod'](keys[-1], **kw)
+                    self._exc_wrapper(
```

Comment:
is really needed to call _exc_wrapper?  ldap.update_entry(entry) should return the same exceptions. IMO it is too complex with _exc_wrapper.